src/main/frontend/src/components/landingPage/InfiniteScrollComponent.js

```diff
@@ -0,0 +1,66 @@
+import InfiniteScroll from "react-infinite-scroll-component";
+import { useEffect, useState } from "react";
+import Item from "./Item";
+import { Row, Col } from "react-bootstrap";
+import { fetchLastChance, fetchNewArrivals } from './ItemService';
+
+function InfiniteScrollComponent(props) {
+  const [items, setItems] = useState([]);
+  const [hasMoreItems, setHasMoreItems] = useState(false);
+  const [page, setPage] = useState(0);
+
+  useEffect(async () => {
+    let data = "";
+
+    if (props.load === "newarrivals") {
```

Comment:
Lets make new file called constants.js and define const LANDING_PAGE_TAB_VALUES = {
NEW_ARRIVALS: "NEW_ARRIVALS",
LAST_CHANCE: "LAST_CHANCE" } and use here as `if load === LANDING_PAGE_TAB_VALUES.NEW_ARRIVALS` instead plain text